itests/org.openhab.core.thing.tests/src/main/java/org/openhab/core/thing/internal/ThingRegistryOSGiTest.java

```diff
@@ -237,7 +237,11 @@ public boolean supportsThingType(@NonNull ThingTypeUID thingTypeUID) {
         waitForAssert(() -> {
             assertTrue(thingResultWrapper.get() != null);
         });
-        assertThat(thing, is(thingResultWrapper.get()));
+
+        assertThat(this, is(notNullValue()));
```

Comment:
Can you shortly explain this assertion?

Reply:
I'll fix that because it should have checked the `thing` instead of `this` so the compiler will know it is null safe but will still fail if null . :+1: